{
    "affected": [
        {
            "package": {
                "ecosystem": "Alpaquita:23",
                "name": "python3",
                "purl": "pkg:apk/alpaquita/python3?arch=source&distro=23"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "introduced": "3.11.3-r0"
                        },
                        {
                            "fixed": "3.11.11-r1"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        },
        {
            "package": {
                "ecosystem": "Alpaquita:stream",
                "name": "python3",
                "purl": "pkg:apk/alpaquita/python3?arch=source&distro=stream"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "introduced": "3.11.4-r0"
                        },
                        {
                            "fixed": "3.12.9-r0"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        }
    ],
    "id": "BELL-CVE-2025-0938",
    "modified": "2025-02-14T15:03:16.542908Z",
    "published": "2025-02-01T05:57:04.668057Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://docs.bell-sw.com/security/cves/CVE-2025-0938"
        }
    ],
    "schema_version": "1.6.7",
    "upstream": [
        "CVE-2025-0938"
    ]
}
